site/data/charts/check-for-updates.json

```diff
@@ -0,0 +1,72 @@
+{
+  "title": "Check for Updates",
+  "description": "The number of Check for Update pings received per month. From 2.9.0 the suffix 'd' indicates ZAP is running as a daemon rather than the desktop.",
```

Comment:
nit, `pings` → `requests` or the other way around above, to use the same wording.

Reply:
Just working on the job to update the data automatically. I can fix this in that :)